src/replica/bulk_load/replica_bulk_loader.cpp

```diff
@@ -534,7 +534,30 @@ void replica_bulk_loader::download_sst_file(const std::string &remote_dir,
                                             int32_t file_index,
                                             dist::block_service::block_filesystem *fs)
 {
-    const file_meta &f_meta = _metadata.files[file_index];
+    if (_status != bulk_load_status::BLS_DOWNLOADING) {
+        LOG_WARNING_PREFIX("Cancel download_sst_file task, because bulk_load local_status is {}. "
+                           "local_dir: {} , file_index is {}.",
```

Comment:
```suggestion
        LOG_WARNING_PREFIX("Cancel downloading sst files, because bulk load local status is {}. "
                           "local_dir: {}, file_index is {}.",
```

Reply:
And why mention local_dir and file_index, is it necessary?